.github/workflows/tests_lgpu_python.yml

```diff
@@ -187,7 +187,7 @@ jobs:
         run: |
           cd main/
           DEVICENAME=`echo ${{ matrix.pl_backend }} | sed "s/_/./g"`
-          pl-device-test --device ${DEVICENAME} --skip-ops --shots=20000 $COVERAGE_FLAGS --cov-append
+          pl-device-test --device ${DEVICENAME} --skip-ops --shots=40000 $COVERAGE_FLAGS --cov-append
```

Comment:
Would you like to revert this change here and in other CI recipes? Python tests are more stable now with less number of stochastic failures.

Reply:
Will do.